CHANGES.rst

```diff
@@ -54,6 +54,9 @@ astropy.modeling
 
 - A ``deepcopy()`` method was added to models. [#6515]
 
+- Added ``is_separable`` function to modeling totest the
```

Comment:
to test

Reply:
Also should this say 'separable property' instead of 'is_separable function'?

Reply:
Now I am confused. I thought `is_separable` is a function that is defined in the new separable module?

Reply:
It is true that this PR adds also `Model.separable` property which is only of interest when one writes a new model. The important functionality from a user point is the function `is_separable`. For completeness I added the `separable` property to the change log as well.